client/components/Cart.js

```diff
@@ -26,27 +26,27 @@ class Cart extends React.Component {
         {cart.map(item => (
           <div key={item.pickle.id}>
             <Link to={`/pickles/${item.pickle.id}`}> {item.pickle.title} </Link>
-            <p>Quantity: {item.quantity}</p>
+            <p>Quantity: {item.qty}</p>
```

Comment:
Maybe we can move QTY inbetween the "remove one"  & "add one" section so the + - is separated by the quantity

Reply:
Yes I like that idea! I'll fix that :)